common/smoke-test/Initialize-SmokeTests.ps1

```diff
@@ -145,8 +145,10 @@ function Update-SampleDependencies {
   $packageSpec = (Get-Content -Path "$($sample.SamplesDirectory)/package.json"
     | ConvertFrom-Json -AsHashtable)
 
+  # some @azure/ packages are not managed in our repo thus should not be overriden
+  $packagesToSkip = @("@azure/msal-node")
```

Comment:
Currently this is the only package that causes problems, so I am in-lining it here.  If there are more in the future we can then try to have a better way to manage the list.